spec/models/invoice_item_spec.rb

```diff
@@ -7,4 +7,8 @@
     it { should belong_to(:item) }
 
   end
+
+  describe 'enums' do
+    it { should define_enum_for(:status).with_values([:pending, :packaged, :shipped])}
+  end
 end
```

Comment:
Want to ask if we need further testing for the inputs or if enums force the three possibilities.